docs/versionhistory.rst

```diff
@@ -8,6 +8,9 @@ This library adheres to `Semantic Versioning 2.0 <https://semver.org/#semantic-v
 - Fixed ``warn_on_error()`` not showing where the type violation actually occurred
 - Fixed local assignment to ``*args`` or ``**kwargs`` being type checked incorrectly
 - Fixed ``TypeError`` on ``check_type(..., None)``
+- **BACKWARD INCOMPATBILE** Made ``check_type`` independent of ``typeguard.config``, it
+  now has the same default configuration values as ``TypeCheckConfiguration`` and is
+  given the ability to accept configuration options as keyword arguments.
```

Comment:
Backward incompatible changes should be displayed first.
```suggestion
- **BACKWARD INCOMPATIBLE** Made ``check_type()`` independent of ``typeguard.config``, it
  now has the same default configuration values as ``TypeCheckConfiguration`` and is
  given the ability to accept configuration options as keyword arguments.
- Fixed ``warn_on_error()`` not showing where the type violation actually occurred
- Fixed local assignment to ``*args`` or ``**kwargs`` being type checked incorrectly
- Fixed ``TypeError`` on ``check_type(..., None)``
```

Reply:
Fixed.